doc/source/developer_guide/importers_module_example.py

```diff
@@ -0,0 +1,251 @@
+# -*- coding: utf-8 -*-
+"""
+One line description of this module. E.g. "Awesome Bureau of Composites importers."
+
+Here you can write a more extensive description (optional). For example, describe the
+readers that are implemented (e.g. Grib2 and Netcdf) and any other thing that you
+consider relevant.
+"""
+
+# Here import your libraries
+import numpy as np
+
+### Uncomment the next lines if pyproj is needed for the importer.
+# try:
+#     import pyproj
+#
+#     PYPROJ_IMPORTED = True
+# except ImportError:
+#     PYPROJ_IMPORTED = False
+
+from pysteps.decorators import postprocess_import
+
```

Comment:
```suggestion
from pysteps.exceptions import MissingOptionalDependency

```

Add this in case optional dependencies are needed? Maybe we could include as commented line?